examples/osggeometry/osggeometry.cpp

```diff
@@ -543,9 +545,10 @@ class MyTransformCallback : public osg::NodeCallback
 
         float               _angular_velocity;
 
-};
-
+}; //class MyTransformCallback
 
+/// Create a quad that sits behind the geometry from createScene()
+/// and shows the image from the OpenGL docs that geometry duplicates.
```

Comment:
I agree that disabling lighting is not related to the functionality of `osg::Geometry`.  I am thinking back to about two years ago, when I was just starting to learn OSG.  I remember being very confused, as a raw newbie, about what I now know is the functionality split between `Geometry`, on the one hand, and `osgViewer::Viewer` plus the automatic light source indicated by `osg::View::LightingMode`, on the other.  I think removing lighting from this example would have reduced my confusion, or at least told me there was more going on that I needed to look into.

My apologies for not being more descriptive in the PR text.  Regardless, thanks very much for your consideration!